test/integration/offline.js

```diff
@@ -49,7 +49,7 @@ describe('offline', function () {
     complete.docs_written.should.not.equal(0)
     complete = await cozy.offline.replicateFromCozy(DOCTYPE)
     complete.docs_written.should.equal(0)
-  })
+  }).timeout(3 * 1000)
```

Comment:
On my computer this test take 2,4 secondes. Default timeout is 2 secondes so I update it.